lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -1369,7 +1369,7 @@ def _verify_html_file_report(self, report_store, expected_data):
         with report_store.storage.open(report_path) as html_file:
             html_file_data = html_file.read()
             for data in expected_data:
-                self.assertIn(data, html_file_data)
+                self.assertIn(data, html_file_data.decode('utf-8'))
```

Comment:
Maybe move the decode operation outside the loop so it doesn't get repeated for each iteration?